packaging/deb/control/postinst

```diff
@@ -26,11 +26,8 @@ case "$1" in
             usermod -a -G systemd-journal "$GRAFANA_AGENT_USER"
         fi
 
-        if [ ! -d /var/lib/grafana-agent ]; then
-          mkdir -p /var/lib/grafana-agent
-          chown $GRAFANA_AGENT_USER:$GRAFANA_AGENT_GROUP /var/lib/grafana-agent
-          chmod 770 /var/lib/grafana-agent
-        fi
+        chown $GRAFANA_AGENT_USER:$GRAFANA_AGENT_GROUP /var/lib/grafana-agent
```

Comment:
I thought of suggesting this last time around, but thought that if a user had manually changed things, they'd have some reason for doing so.

Should we leave ownership/permissions if the folder already exists?

Reply:
Is there a way we can give it permissions by default? It's set to the home directory of the user so it's being created with unpredictable permissions unless we force it (ditto with the config file)

Reply:
I'm not sure. I don't want to block things, let's go ahead with this, should be fine.